test/e2e/traceflow_test.go

```diff
@@ -0,0 +1,278 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"github.com/stretchr/testify/require"
+	networkingv1 "k8s.io/api/networking/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"testing"
+	"time"
+
+	"github.com/vmware-tanzu/antrea/pkg/apis/ops/v1alpha1"
+	"github.com/vmware-tanzu/antrea/pkg/client/clientset/versioned"
+)
+
+type testcase struct {
+	tf              *v1alpha1.Traceflow
+	expectedPhase   v1alpha1.TraceflowPhase
+	expectedResults []v1alpha1.NodeResult
+}
+
+func TestTraceflow(t *testing.T) {
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	node1 := workerNodeName(1)
+	node2 := workerNodeName(2)
+
+	pod0 := data.createPod(node1, t)
```

Comment:
consider using `createTestBusyboxPods` to avoid having to define yet another Pod creation function and to have less verbose tests

it is defined in fixtures.go

Reply:
Updated.